iree/test/model_benchmarks/CMakeLists.txt

```diff
@@ -0,0 +1,174 @@
+iree_mlir_benchmark_suite(
+  MODULE_NAMES
```

Comment:
I wouldn't say that I love the duplication here just to vary how it is run. Wdyt about putting these in a for loop that just varies the driver flags? Or just set a local variable of MOBILENET_SOURCE_PARAMS and just rely on CMake being CMake and referencing it in the iree_mlir_benchmark_suite?

```
iree_mlir_benchmark_suite(${MOBILENET_SOURCE_PARAMS}
  ... additional ...
)
```

Reply:
Yeah. This is really the tricky part. Ideally we should have a set of models and a set of benchmark configurations (including translation/runtime flags, benchmark modes, etc.) and just get all their combinations. But that isn't really possible for now (and probably for quite some time into the future) given not all models works with all configurations. Going down that route right now means there will be too many "failing combinations" to explicitly exclude. So I'm doing this direct coupling between model specification and benchmark configuration specification to match the current status. 

But I see your point about duplication. I think I can do another level of commonality here by defining a bunch of list ahead for those models that always work for all benchmark configurations. Hopefully all models fall into this category so we just need to register it at one place instead of duplicating for all rules. For those that cannot uniformly work, they will still need to be duplicated per case. This can also serve as an indicator of whether we can totally decouple the model list and benchmark configuration list and go towards that eventually.